README.md

```diff
@@ -36,16 +36,18 @@ To determine the line of demarcation between a general mid-level ontology for cy
 1. Controls
 1. Resilience
 
-The above list is by no means all inclusive. The gistCyber ontology was created to enable 
+The above list is by no means all inclusive. The gistCyber ontology was created to enable
 semantic interoperability between these many sub-domains of cybersecurity.
 
 ## The Relationship between gistCyber and other Ontologies
 
 ### The gistCyber ontology has dependencies
-1. Basic Formal Ontology
+1. Basic Formal Ontology (BFO)
 1. gist
 1. gistComputing
 
+gistCyber imports both gist and BFO through [gistBFO](https://github.com/semanticarts/gistBFO), a BFO-compatible version of gist. This has allowed for gistCyber to be easily extended from gist, all the while maintaining alignment with BFO.
+
```

Comment:
@rhohimer I've started with a short note, but if there is anything you think would be helpful to expand on (e.g., DoD-IC mandate), happy to build it out as needed.